fenix/app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -655,6 +656,11 @@ class HomeFragment : Fragment() {
                     text = it.name,
                     start = DrawableMenuIcon(
                         drawable = it.icon.toDrawable(resources),
+                        tint = if (it.type == SearchEngine.Type.APPLICATION) {
+                            requireContext().getColorFromAttr(R.attr.textPrimary)
```

Comment:
```suggestion
                            requireContext().getColorFromAttr(R.attr.iconPrimary)
```
We should use the correct variable here. textPrimary is for text whereas we want the icon variable here.

Reply:
let's merge it as it is, I will file a bug to add the attribute. currently there is no `iconPrimary` attribute, and the majority is the icons in the xml are using `android:fillColor="?attr/textPrimary"` as a filler.

Reply:
Yes, this makes sense to me. Historically, we have been using `textPrimary` for both text and icons before the introduction of the `iconPrimary` with the design system. So, you will see in almost everywhere that we're using `textPrimary` for the icon color.